src/js/model.js

```diff
@@ -34,57 +37,68 @@ export const mixerList = [
 
 // 3D model
 const Model = function (wrapper, canvas) {
-    const useWebGLRenderer = this.canUseWebGLRenderer();
+    // Configure model detail level (1-10, where 1 is lowest detail and 10 is highest)
+    this.detailTolerance = 10; // Default value, can be modified
+
+    this.useWebGLRenderer = this.canUseWebGLRenderer();
 
     this.wrapper = wrapper;
     this.canvas = canvas;
 
-    if (useWebGLRenderer) {
-        this.renderer = new THREE.WebGLRenderer({ canvas: this.canvas[0], alpha: true, antialias: true });
+    if (this.useWebGLRenderer) {
+        this.renderer = new THREE.WebGLRenderer({
+            canvas: this.canvas[0],
+            alpha: true,
+            antialias: true, // Disable antialiasing for performance
+        });
     } else {
-        this.renderer = new THREE.CanvasRenderer({ canvas: this.canvas[0], alpha: true });
+        console.log("Starting in low performance rendering mode");
+        this.renderer = new CanvasRenderer({
+            canvas: this.canvas[0],
+            alpha: true,
+        });
     }
 
-    // initialize render size for current canvas size
     this.renderer.setSize(this.wrapper.width(), this.wrapper.height());
 
     // load the model including materials
-    let model_file = useWebGLRenderer ? mixerList[FC.MIXER_CONFIG.mixer - 1].model : "fallback";
+    let model_file = mixerList[FC.MIXER_CONFIG.mixer - 1]?.model;
 
     // Temporary workaround for 'custom' model until akfreak's custom model is merged.
-    if (model_file == "custom") {
+    if (model_file === undefined || model_file == "custom") {
```

Comment:
This is the safe implementation
```suggestion
    if (model_file === undefined || model_file === "custom") {
```

But looking at mixerList fallback was not needed anyway :smirk: 
```suggestion
    if (model_file === "custom") {
```